src/routes/(console)/project-[region]-[project]/functions/create-function/deploy/+page.ts

```diff
@@ -19,10 +30,112 @@ export const load: PageLoad = async ({ url, params, parent }) => {
     }
 
     const envParam = url.searchParams.get('env');
-    const envKeys = envParam ? envParam.split(',').map((key: string) => key.trim()) : [];
+
+    // Parse env vars - supports KEY or KEY=value format
+    const envVars: Array<{ key: string; value: string }> = envParam
+        ? envParam.split(',').map((entry: string) => {
+              const trimmed = entry.trim();
+              const eqIndex = trimmed.indexOf('=');
+              if (eqIndex === -1) {
+                  return { key: trimmed, value: '' };
+              }
+              return {
+                  key: trimmed.substring(0, eqIndex),
+                  value: trimmed.substring(eqIndex + 1)
+              };
+          })
+        : [];
 
     const runtime = url.searchParams.get('runtime');
 
+    // Quick mode - create function and redirect directly to function page
+    const quickMode = url.searchParams.get('quick') === 'true';
+
+    if (quickMode) {
+        try {
+            const runtimeParam = runtime || 'node-18.0';
```

Comment:
mm, not sure about the default value of `node-18`. IIRC self-hosted has 22 enabled by default? whatever the value on SH side can be used here.